composer.json

```
@@ -26,20 +26,19 @@
     "pimcore/object-merger": "^3.0",
     "pimcore/pimcore": "^10.5.6",
```

```suggestion
    "pimcore/pimcore": "^11",
```
in case we drop support for 10.5 right? otherwise it should be all like: 
`"symfony/dependency-injection": "^5.3.3 || ^6.1"`